### DISCUSSION TITLE
UI - Bag Bars & Minimap Buttons

### DISCUSSION THREAD

**Discussion by GiscoWarlock** [runelite/runelite#15154] [Q&A] [answered]
Would like the bag bars detached with the option to have them horizontal and moveable via Alt key like the rest of the UI.
The minimap buttons should as well have an option to detach and move individually.
From [this](https://sketchy.questionable.link/5zt4_z2w8.png) to [this](https://sketchy.questionable.link/5zt3Ph8Xl.jpeg).

Thank you very much. :)

**Comment by Nightfirecat** [comment, marked as answer]
#14582 is open for allowing you to make the bars moveable and #8516 is open for allowing them to be horizontal.
